index.html

```diff
@@ -0,0 +1,235 @@
+                            <li><img src="Vector1.png" alt=""></li>
+                            <li><img src="Vector3.png" alt=""></li>
+                        </ul>
+                    </div>
+                    <section class="bottom-images">
+                        <div id="elipse12">
+                            <img src="Ellipse12.png" alt="">
+                        </div>
+                        <div id="elipse13">
+                            <img src="Ellipse13.png" alt="">
+                        </div>
+                        <div id="elipse15">
+                            <img src="Ellipse15.png" alt="">
+                        </div>
+                        <div id="elipse4">
+                            <img src="Ellipse4.png" alt="">
+                        </div>
+                        <div id="rectangle5">
+                            <img src="Rectangle5.png" alt="">
+                        </div>
+                    </section>
+                </section>
+            </div>
+    </article>
+        
+        <section class="work-card">
+            <div class="work1">
+                <img src="Snapshoot.png" alt="">
+               
+                <h2>Tonic</h2>
+                <div id="frame">
+                    <div id="text1"><h5>CANOPY</h5></div>
+                    <div id="counter"></div>
+                    <div id="role"><h5>Back End Dev</h5></div>
+                    <div id="counter1"></div>
+                    <div id="text"><h5>2015</h5></div>
+                </div>
+                <div class="primary-text">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="tags">
+                    <div id="tag1"><div id="html">html</div></div>
+                    <div id="tag2"><div id="css">css</div></div>
+                    <div id="tag3"><div id="javascript">javascript</div></div>
+                </div>
+                <div class="Actions">
+                    <div id="see-project">
+                        <div><h4>See Project</h4></div>
+                    </div>
+                </div>
+            </div>
+            <div class="work2">
+                <img src="SnapshootPortfolio.png" alt="">
+               
+                <h2>Multi-Post Stories</h2>
+                <div id="work2frame">
+                    <div id="work2text1"><h5>CANOPY</h5></div>
+                    <div id="work2counter"></div>
+                    <div id="work2role"><h5>Back End Dev</h5></div>
+                    <div id="work2counter1"></div>
+                    <div id="work2text"><h5>2015</h5></div>
+                </div>
+                <div class="work2primary-text">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="work2tags">
```

Comment:
- [ ] As suggested by the previous code reviewer,  This is a list and it would be semantically more appropriate to use [unordered lists ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul) i.e `<ul>` with nested `li` instead of <div>. Kindly make changes to everywhere affected in this file.